Flow.Launcher/Languages/en.xaml

```diff
@@ -231,4 +232,40 @@
     <system:String x:Key="update_flowlauncher_update_files">Update files</system:String>
     <system:String x:Key="update_flowlauncher_update_upadte_description">Update description</system:String>
 
+    <!--  Welcome Window  -->
+    <system:String x:Key="Skip">Skip</system:String>
+    <system:String x:Key="Welcome_Page1_Title">Welcome to Flow Launcher</system:String>
+    <system:String x:Key="Welcome_Page1_Text01">Hello, It's your first time to run a Flow Launcher!</system:String>
+    <system:String x:Key="Welcome_Page1_Text02">Before starting, proceed with a simple setting. Surely, You can skip. Please choose the language you use.</system:String>
```

Comment:
Before starting, this wizard will assist in setting up Flow Launcher. You can skip this if you wish. Please choose a language